lib/ruby/stdlib/resolv.rb

```diff
@@ -743,66 +739,49 @@ class UnconnectedUDP < Requester # :nodoc:
         def initialize(*nameserver_port)
           super()
           @nameserver_port = nameserver_port
-          @mutex = Thread::Mutex.new
-          @initialized = false
-        end
-
-        def lazy_initialize
```

Comment:
just for confirmation, this was backported to ruby 2.3, 2.4 and 2.6?
isn't 2.5 also affected?
see https://bugs.ruby-lang.org/issues/14571 and https://github.com/jruby/jruby/pull/5074

Reply:
Yes, this was originally filed against 2.5 it appears and backported as far as 2.3.

Reply:
yeah, but this change reverts the fix. Maybe MRI guys forgot to backport it?